openslides/motions/config_variables.py

```diff
@@ -151,6 +151,19 @@ def get_config_variables():
         group='Motions',
         subgroup='Supporters')
 
+    # Comments
+
+    yield ConfigVariable(
+        name='motions_comments',
+        default_value='Comment',
+        input_type='text',
+        label='Comment fields for motions',
+        help_text='Use comma separated list of field names for public '
+                  'fields or use special JSON. See manual for syntax.',
```

Comment:
Please add this short example to the help text (we should use it until we replace it by an improved UI to select public/private via checkbox): 
```
[{"name": "foo", "public": false},{"name": "bar", "public": true}]
```